app/templates/pay.hbs

```diff
@@ -7,9 +7,17 @@
     </h1>
 
     {{#if this.activeDiscountForYearlyPlan.isFromSignup}}
-      <PayPage::SignupDiscountNotice @discount={{this.activeDiscountForYearlyPlan}} class="mt-8" />
+      <PayPage::SignupDiscountNotice
```

Comment:
Let's split this into two components - something like `SignupDiscountNotice` and `ExperimentalSignupDiscountNotice`, would be a bit cleaner to change in the future too + feature flag behaviour is better shown at the top-level than deeply nested somewhere